sys/crypto/doc.txt

```diff
@@ -25,6 +25,12 @@
  * While you can use the ciphers functions directly, you should resort to
  * the generic API for block ciphers whenever possible.
  *
+ * Additionally you need to set a CFLAG for each cipher you want to use in your Makefile:
+ *  * AES-128: CFLAGS += -DCRYPTO_AES
+ *  * 3DES:    CFLAGS += -DCRYPTO_THREEDES
+ *  * Twofish: CFLAGS += -CRYPTO_TWOFISH
+ * This sets the appropriate buffer sizes for cipher operations.
```

Comment:
you mean say that it sets the buffer size of the `cipher_context_t` used by the ciphers for en-/de-cryption operations